src/components/Form/CheckAclPermsDocContainerForm.jsx

```diff
@@ -1,5 +1,13 @@
-import React, { useContext } from 'react';
+// React Imports
+import React, { useContext, useState } from 'react';
+// Solid Imports
 import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
```

Comment:
Until they're used in this form, I don't think we need to import any MUI components here.